lib/integrate.js

```diff
@@ -19,27 +19,7 @@
 var fs = require('fs');
 var path = require('path');
 var gutil = require('gulp-util');
-
-function copyFile(src, dest) {
-  return new Promise(function(resolve, reject) {
-    var srcStream = fs.createReadStream(src);
-
-    srcStream.pipe(fs.createWriteStream(dest));
-
-    srcStream.on('error', reject);
-
-    srcStream.on('end', function() {
-      console.log(gutil.colors.blue(
-        'Your app needs to load the script offline-manager.js in order to register the service\n' +
-        'worker that offlines your app. To load the script, add this line to your app\'s HTML\n' +
-        'page(s)/template(s):\n\n' +
-        '\t<script src="' + dest + '"></script>'
```

Comment:
We should still show this message, even if we aren't using this custom *copyFile* method anymore.